src/components/Sidebar/Sidebar.js

```diff
@@ -141,67 +153,75 @@ export class Sidebar extends PureComponent<Props, State> {
-                    style={{
-                      ...style,
-                      background: COLORS.transparentWhite[700],
-                      borderRadius: '6px',
-                    }}
-                    className="thumb-vertical"
-                  />
-                )}
-                renderTrackHorizontal={props => (
-                  <div
-                    {...props}
-                    style={{
-                      display: 'none',
-                    }}
-                  />
-                )}
-              >
-                <IntroductionBlurb
-                  isVisible={!finishedOnboarding && introSequenceStepIndex >= 1}
-                />
-                <Projects offset={`${interpolated.firstProjectPosition}px`}>
-                  {projects.map(project => (
-                    <Fragment key={project.id}>
-                      <Tooltip
-                        title={project.name}
-                        position="right"
-                        transitionFlip={false}
-                      >
-                        <SidebarProjectIcon
-                          size={SIDEBAR_ICON_SIZE}
-                          id={project.id}
-                          name={project.name}
-                          color={project.color}
-                          iconSrc={project.icon}
-                          isSelected={
-                            finishedOnboarding &&
-                            project.id === selectedProjectId
-                          }
-                          handleSelect={() => selectProject(project.id)}
-                        />
-                      </Tooltip>
-                      <Spacer size={18} />
-                    </Fragment>
-                  ))}
-                  <AddProjectButton
-                    size={SIDEBAR_ICON_SIZE}
-                    onClick={createNewProjectStart}
-                    isVisible={
-                      finishedOnboarding || introSequenceStepIndex >= 2
-                    }
-                    isOnline={isOnline}
-                  />
-                </Projects>
-              </ScrollbarOnlyVertical>
+              <IntroductionBlurb
+                isVisible={!finishedOnboarding && introSequenceStepIndex >= 1}
+              />
+              <DragDropContext onDragEnd={this.onDragEnd}>
+                <Scrollbars autoHide>
+                  <Droppable droppableId="droppable">
+                    {provided => (
+                      <div ref={provided.innerRef}>
+                        <Projects
+                          offset={`${interpolated.firstProjectPosition}px`}
+                        >
+                          {projects.map((project, index) => (
+                            <Draggable
+                              key={project.id}
+                              draggableId={project.id}
+                              index={index}
+                              disableInteractiveElementBlocking
```

Comment:
I think we should disable dragging if there's only one project since there would be no need to re-arrange in that case. Can do it with `isDragDisabled`:

```suggestion
                              disableInteractiveElementBlocking
                              isDragDisabled={projects.length === 1}
```

Reply:
Good idea. I've added it. I also added `pointer-events: none` to the sidebar image because with-out it there would be a "ghost" image draggable.

**To the offset issue:** Yes, would be great if you could have a look. I still don't know why this happens. I think it's not related to `React-custom-scrollbars` because the same happens if I'm using css overflow `scroll` with out custom scrollbars. Could be related to React-Beautiful-Dnd auto-scroll behaviour but I need to do more debugging.